### PR TITLE
[Form] Check for Intl availibility

### PR DESCRIPTION
**Pull request by ro0NL**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no     
| Deprecations? | no
| Tests pass?   | yes    
| Fixed tickets | #...   
| License       | MIT
| Doc PR        | symfony/symfony-docs#... 

Same as #28513 for the form component

### CODE REVIEW

File: src/Symfony/Component/Form/Extension/Core/Type/CountryType.php [hunk 1]
```diff
@@ -41,6 +42,10 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
             'choice_loader' => function (Options $options) {
+                if (!class_exists(Intl::class)) {
+                    throw new LogicException('The "symfony/intl" component is required to use the Country type.');
+                }
```
Comment (yceruto) [inline src/Symfony/Component/Form/Extension/Core/Type/CountryType.php]: Could we move this to `IntlCallbackChoiceLoader` instead?

Comment (ro0NL) [reply src/Symfony/Component/Form/Extension/Core/Type/CountryType.php]: the message would be less obvious.. currently it hints `Country type` etc.

File: src/Symfony/Component/Form/Extension/Core/Type/CountryType.php [hunk 2]
```diff
@@ -41,6 +42,10 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
             'choice_loader' => function (Options $options) {
+                if (!class_exists(Intl::class)) {
+                    throw new LogicException('The "symfony/intl" component is required to use the Country type.');
```
Comment (OskarStark) [inline src/Symfony/Component/Form/Extension/Core/Type/CountryType.php]: I would prefer `LanguageType:class`, to get the FQCN here, same for the other exceptions

**Review by chalasr — APPROVED** [review]

**Review by xabbuh — APPROVED** [review]
but @OskarStark's suggestion to enhance the message sounds good to me

File: src/Symfony/Component/Form/Extension/Core/Type/CountryType.php (hunk 2 above)
Comment (ro0NL) [reply src/Symfony/Component/Form/Extension/Core/Type/CountryType.php]: Done.

**Review by fabpot — APPROVED** [review]

**Pull request merged**